src/eventcontroller/DefaultEventController.ts

```diff
@@ -15,7 +22,7 @@ import EventName from './EventName';
 import flattenEventAttributes from './flattenEventAttributes';
 import MeetingHistoryState from './MeetingHistoryState';
 
-export default class DefaultEventController implements EventController {
+export default class DefaultEventController implements EventController, Destroyable {
```

Comment:
Can we call out a few sentences of implementation details? In particular the usage of `EventReporter`

Reply:
Added a few comments, let me know if you think there's any parts that are confusing that I should clarify.